services/docker-compose.local.yml

```diff
@@ -109,9 +109,11 @@ services:
 
   webserver:
     environment:
-      - GUNICORN_CMD_ARGS=--timeout=0
-      - SC_BOOT_MODE=${SC_BOOT_MODE:-default}
-      - REST_SWAGGER_API_DOC_ENABLED=1
+      &webserver_environment_local
+      SC_BOOT_MODE: ${SC_BOOT_MODE:-default}
```

Comment:
default value. Shall we avoid them in here?

Reply:
yes, but one step at a time. I am doing that in a separate PR https://github.com/ITISFoundation/osparc-simcore/pull/5776